bears/community/AuthorsBear.py

```diff
@@ -0,0 +1,23 @@
+from dependency_management.requirements.GemRequirement import GemRequirement
+from dependency_management.requirements.GoRequirement import GoRequirement
+from coalib.bearlib.abstractions.Linter import linter
+
+
+@linter(executable='maintainer',
+        global_bear=True,
+        output_format='regex',
+        output_regex=r'(?P<severity>Error|Warning) \d+ in .+ line '
+                     r'(?P<line>\d+): (?P<message>.*)'
```

Comment:
@gaocegege Is there really output like this from `maintainer`? As mentioned in https://github.com/coala/coala-bears/issues/2106#issuecomment-344021732 I really don't think this should be a `linter` bear

Reply:
Yeah, I forgot to update it.